test/runtests.jl

```diff
@@ -38,6 +38,7 @@ end
 
 # list of tests
 testfiles = [
+  "colspec.jl",
   "distributions.jl",
```

Comment:
Maybe these two lines should be flipped to follow the same order of the main TableTransforms.jl includes?

Reply:
OK

Reply:
Made in [084a6ef](https://github.com/JuliaML/TableTransforms.jl/pull/41/commits/084a6efea3cbf08d265c257e0b178ef8d545e9b9)